### PR TITLE
[Task#xxx] fix seeder data

### PR DESCRIPTION
**Pull request by NamBZ**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## WHAT this PR does?
Sửa lại data seeder (Chỉ seed User, Category và Product)

## Notes Impacted Areas:
- N/A.

## Attachments:
image

### CODE REVIEW

**Review by quyensama — APPROVED** [review]

**Comment by NamBZ** [comment]
ready

File: database/seeders/DatabaseSeeder.php [hunk 1]
```diff
@@ -17,10 +17,10 @@ public function run()
...
             CategorySeeder::class,
             ProductSeeder::class,
-            OrderSeeder::class,
-            OrderItemSeeder::class,
-            RatingSeeder::class,
-            ShippingSeeder::class,
+            //OrderSeeder::class,
+            //OrderItemSeeder::class,
+            //RatingSeeder::class,
+            //ShippingSeeder::class,
```
Comment (ghost) [inline database/seeders/DatabaseSeeder.php]: sao lại comment ntn????

File: database/data/products.json [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
... cô đơn hơn cả một kẻ đắm tàu sống sót trên bè giữa đại dương. Thế nên các bạn cứ tưởng tượng tôi đã ngạc nhiên làm sao, khi ánh ngày vừa rạng, thì một giọng nói nhỏ nhẹ lạ lùng đã đánh thức tôi. Giọng ấy nói:<br>\r\n“Ông làm ơn… vẽ cho tôi một con cừu!”<br>\r\n- Trích \"Hoàng tử bé\"<p>Giá sản phẩm trên Tiki đã bao gồm thuế theo luật hiện hành. Bên cạnh đó, tuỳ vào loại sản phẩm, hình thức và địa chỉ giao hàng mà có thể phát sinh thêm chi phí khác như phí vận chuyển, phụ phí hàng cồng kềnh, thuế nhập khẩu (đối với đơn hàng giao từ nước ngoài có giá trị trên 1 triệu đồng).....<\/p>","description":"<table><tbody><tr><td>Công ty phát hành<\/td><td>Nhã Nam<\/td><\/tr><tr><td>Ngày xuất bản<\/td><td>2019-05-01 00:00:00<\/td><\/tr><tr><td>Kích thước<\/td><td>15 x 23 cm<\/td><\/tr><tr><td>Dịch Giả<\/td><td>Trác Phong<\/td><\/tr><tr><td>Loại bìa<\/td><td>Bìa mềm<\/td><\/tr><tr><td>Số trang<\/td><td>102<\/td><\/tr><\/tbody><\/table>","image":"https:\/\/salt.tikicdn.com\/cache\/400x400\/ts\/product\/5c\/e7\/68\/26838e18d7f96d562d828980520019d2.jpg","slug":"hoang-tu-be-tai-ban-2019","quantity":"148","sold":"0","retail_price":"53500","original_price":"75000","avg_rate":"0","category_id":"35"},
```
Comment (ghost) [inline database/data/products.json]: xuống dòng nha.

**Comment by NamBZ** [comment]
ready

**Pull request merged**